libraries/ESP_NOW/src/ESP32_NOW.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "esp_wifi_types.h"
+#include "Print.h"
+#include "esp_now.h"
+#include "MacAddress.h"
+
+class ESP_NOW_Peer {
+private:
+  uint8_t mac[6];
+  uint8_t chan;
+  wifi_interface_t ifc;
+  bool encrypt;
+  uint8_t key[16];
+
+protected:
+  bool added;
+  bool add();
+  bool remove();
+  size_t send(const uint8_t * data, int len);
+
+public:
+  const MacAddress BROADCAST_ADDR(0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF);
```

Comment:
Just for users to easily have the broadcast address (ff:ff:ff:ff:ff:ff) when needed. Most of the IDF examples that use it just define it as a static variable in the file. IMO it will make it easier for users to use it  and for them to learn that this address is the broadcast one.

Reply:
@me-no-dev Do you think this would be better done in a per sketch basis as it might not be used by everyone ?

Reply:
This is does not seem to be the right way :) just something is off, like ESP_NOW does not yet accept MacAddress as argument. Add a broadcast example as-is and we can discuss further.

Reply:
It works fine as MacAddress have `operator const uint8_t*() const`. You can check it in the current example.

Reply:
Ok, maybe move it to the ESP_NOW class, so you init a peer with `SomePeer(ESP_NOW::BROADCAST_ADDR)`. Can you add example with broadcasts? A basic packet based example?

Reply:
I will. Working on it now.